### PR TITLE
Approve renovate PRs automatically

### PR DESCRIPTION
**Pull request by ab77**
* simplify renovate-config(s)
* [renovate](https://github.com/product-os/renovate-config/blob/master/.github/renovate.json#L21-L23) adds `renovate` and `dependencies` labels to each PR
* write privilege required to label PRs

Change-type: minor

### CODE REVIEW

File: policies/approval.yml
Comment (klutchell) [inline policies/approval.yml:72]: I think we need to keep `count: 0` in order to auto-approve with these labels?

File: policies/approval.yml
Comment (klutchell) [inline policies/approval.yml:67]: I would like to keep this `"no-review"` label as well.

File: policies/approval.yml
Comment (klutchell) [inline policies/approval.yml:55]:
```suggestion
  - name: pull request has special labels
```

File: policies/approval.yml
Comment (klutchell) [inline policies/approval.yml:5]:
```suggestion
        - pull request has special labels
```

File: policies/approval.yml
Comment (klutchell) [inline policies/approval.yml:55]: I confirmed that only users with write access can add labels 👍

**Review by klutchell — APPROVED** [review]

File: policies/approval.yml
Comment (ab77) [reply policies/approval.yml:72]: it's the default, do you want me still write it out?

Comment (klutchell) [reply policies/approval.yml:72]: Haha, is it really? No need to write it out, I know how you feel about defaults!

File: policies/approval.yml
Comment (ab77) [reply policies/approval.yml:55]: I hope so, I'd hate to have some random labelling my PRs :)

**Comment by ab77** [comment]
lgtm

**Comment by klutchell** [comment]
Squash first?

**Comment by klutchell** [comment]
LGTM after those commits are squashed

**Comment by klutchell** [comment]
lgtm

**Pull request merged**
